app/javascript/retrospring/features/settings/index.ts

```diff
@@ -10,7 +10,7 @@ export default (): void => {
   themeDocumentHandler();
 
   registerEvents([
-    { type: 'submit', target: document.querySelector('#update_theme'), handler: themeSubmitHandler },
+    { type: 'submit', target: document.querySelector('[action="/settings/theme"]'), handler: themeSubmitHandler },
```

Comment:
Still need to update this, the new route for editing the theme is `/settings/theme/edit`.

Reply:
Restarted the app to see if it maybe was routes only updating on boot, but

![image](https://user-images.githubusercontent.com/1774242/175813467-ea4999e9-bf7b-4a25-b6cb-5a14588e3264.png)

which is the result of

```haml
= bootstrap_form_for(current_user.theme || Theme.new, url: { action: "update" }, html: { id: "update" }, method: :patch) do |f|
```

Reply:
Ah right, I forgot what querySelector does :-)